metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Taxonomy.java

```diff
@@ -75,4 +78,19 @@ public Long getId() {
 
     public String getName() { return name; }
 
+    // Release date control: get the *earliest* release date from all studies which link to this taxonomy
+    @Formula("(SELECT min(study.release_date) FROM taxonomy " +
+            "INNER JOIN sample_taxonomies on taxonomy.id = sample_taxonomies.taxonomies_id " +
+            "INNER JOIN sample on sample_taxonomies.sample_id = sample.id " +
+            "INNER JOIN analysis_samples on sample.id = analysis_samples.samples_id " +
+            "INNER JOIN analysis on analysis_samples.analysis_id = analysis.id " +
+            "INNER JOIN study on analysis.study_id = study.id " +
+            "WHERE sample.id=id)")
+    @JsonIgnore
+    private LocalDate releaseDate;
```

Comment:
I guess the taxonomy and reference sequence will be public. not sure can check with @cyenyxe

Reply:
Yes, none of our archives supports non-public taxonomies and sequences.

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/4b70202f0d05260d4268461280280e1d5296aa10. Taxonomy and WebResource are now always public.